index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script src="js/index.js"></script>
+    <nav id="navbar">
+      <div class="nav-item"><a href="#welcome-section"> About</a></div>
+      <div class="nav-item"><a href="#projects"> Work</a></div>
+      <div class="nav-item"><a href="#contact"> Contact</a></div>
+    </nav>
+
+    <div id="main-div">
+      <div id="welcome-section">
+        <h1>Hey We are Jorge & Kanza</h1>
+        <p>Aspriring Web Developers</p>
+      </div>
+    </div>
+
+    <div id="projects">
+      <section>
+        <h1><span>These are some of our projects</span></h1>
+      </section>
+      <section class="project-tile">
+        <div>
+          <a href="https://codepen.io/freeCodeCamp/full/zNqgVx">
+            <img src="./img/tribute.jpg" alt=" Tribute Website" />
+          </a>
```

Comment:
I think it would be better to use any `h2-h6` headings in the `<section>` tag for better listing and no linter errors.